docker/README.md

```diff
@@ -93,28 +79,14 @@ All Dockerfiles are located in `docker/dockerfiles` folder.
 
 * `zkv-builder.Dockerfile`: create an image with all dependencies needed to compile the node and is used by `my_cargo` script
 * `binary_injected.Dockerfile`: Is mainly used by the scripts and inject one or more binaries in a standard ubuntu image
-* `zkv-node.Dockerfile`: generate a solo node image with a fresh source compilation (leverage on docker layers to create a small docker image)
+* `zkv-docker-compose.yaml`: the cluster definition that runs
```

Comment:
Aren't you deleting this file as well?

Reply:
Yes indeed, messed up with conflicts on rebase, thanks for cathing! Removed